packages/react-core/src/demos/RTL/examples/PaginatedTable.jsx

```diff
@@ -0,0 +1,506 @@
+                    onClick={onDropdownToggle}
+                    icon={<Avatar src={imgAvatar} alt="" />}
+                    isFullHeight
+                  >
+                    {translation.username}
+                  </MenuToggle>
+                )}
+              >
+                <DropdownList>{userDropdownItems}</DropdownList>
+              </Dropdown>
+            </ToolbarItem>
+          </ToolbarContent>
+        </Toolbar>
+      </MastheadContent>
+    </Masthead>
+  );
+
+  return (
+    <React.Fragment>
+      <Page sidebar={sidebar} header={masthead} isManagedSidebar>
+        <PageBreadcrumb>
+          <Breadcrumb aria-label={translation.breadcrumbs.ariaLabel || undefined}>
+            {Object.keys(breadcrumbItems).map((key, idx, arr) => (
+              <BreadcrumbItem key={idx} isActive={arr.length - 1 === idx} to={`${breadcrumbItems[key].url}`}>
+                {translation.breadcrumbs[key]}
+                {breadcrumbItems.length}
+              </BreadcrumbItem>
+            ))}
+          </Breadcrumb>
+        </PageBreadcrumb>
+        <PageSection variant="light">
+          <TextContent>
+            <Text component={TextVariants.h1}>{translation.title}</Text>
+            <Text component={TextVariants.p}>{translation.body}</Text>
+          </TextContent>
+        </PageSection>
+        <PageSection>
+          <Card>
+            {toolbarItems}
+            <Table variant="compact" aria-label={translation.table.ariaLabel}>
+              <Thead>
+                <Tr>
+                  {columns.map((column, columnIndex) => (
+                    <Th key={columnIndex}>{column}</Th>
+                  ))}
+                </Tr>
+              </Thead>
+              <Tbody>
+                {paginatedRows.map((row, rowIndex) => (
+                  <Tr key={rowIndex}>
+                    <>
+                      {Object.entries(row).map(([key, value]) => {
+                        if (key === 'status') {
+                          return (
+                            <Td key={key} width={15} dataLabel="Status">
+                              {renderLabel(value)}
+                            </Td>
+                          );
+                        } else if (key === 'url') {
+                          return (
+                            // Passing dir="rtl" forces truncation at the start of the URL,
+                            // resulting in the unique portion being visible regardless of language
+                            <Td key={key} dataLabel="URL" width={15}>
+                              <a href="#">
+                                <Truncate content={row.url} position={isDirRTL ? 'end' : 'start'} />
+                              </a>
+                            </Td>
+                          );
+                        } else {
+                          return (
+                            <Td key={key} dataLabel={key === 'lastModified' ? 'Last modified' : capitalize(key)}>
```

Comment:
I am not a fan of using undocumented helper functions in our demo code.  Can you create a new function for `capitalize` in the demo code instead.  That way the consumer has access to the code.